source/module.ts

```diff
@@ -1,11 +1,14 @@
 import {Inject, NgModule} from '@angular/core';
 import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 
-const {NgRedux} = require('ng2-redux');
-
 import {Connect} from './connect';
 import {ConnectArray} from './connect-array';
 import {FormStore} from './form-store';
+import { NgRedux } from 'ng2-redux';
+
+export function formStoreFactory(ngRedux: NgRedux<any>) {
```

Comment:
this looks very reasonable to me
I just think that the function doesn't need to be exported since it's only used here internally